src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -57,6 +58,15 @@
 from azure.mgmt.containerservice.models import ManagedClusterAADProfile
 from azure.mgmt.containerservice.models import ManagedClusterAddonProfile
 from azure.mgmt.containerservice.models import ManagedClusterAgentPoolProfile
+from azure.mgmt.containerservice.models import OpenShiftManagedClusterAgentPoolProfile
+from azure.mgmt.containerservice.models import OpenShiftAgentPoolProfileRole
+from azure.mgmt.containerservice.models import OpenShiftManagedClusterIdentityProvider
+from azure.mgmt.containerservice.models import OpenShiftManagedClusterAADIdentityProvider
+from azure.mgmt.containerservice.models import OpenShiftManagedCluster
+from azure.mgmt.containerservice.models import OpenShiftRouterProfile
+from azure.mgmt.containerservice.models import OpenShiftManagedClusterAuthProfile
+from azure.mgmt.containerservice.models import NetworkProfile
```

Comment:
If containerservice is multi-API, all of these imports will need to be changed to use the `get_models` calls like the `vm`, `network`, `storage`, etc. modules.

Reply:
Can we merge this as is ? And refactor in a separated PR with the AKS team ?